templates/CRM/Contact/Form/Search/Criteria/Basic.tpl

```diff
@@ -23,6 +23,13 @@
  | see the CiviCRM license FAQ at http://civicrm.org/licensing        |
  +--------------------------------------------------------------------+
 *}
+{literal}
+<style type="text/css">
+  .adv-search-top-submit span.crm-button {
```

Comment:
Here's the beginnings of a separate stylesheet for search pages: https://github.com/civicrm/civicrm-core/pull/5342